### DISCUSSION TITLE
Meaning of field called "quality"

### DISCUSSION THREAD

**Discussion by AlbertoPresta** [InterDigitalInc/CompressAI#130] [Q&A] [answered]
Hi! 

maybe it is a stupid question, but I wonder what is the physical meaning of the field called "quality".
When You use this repo to train "bmshj2018_factorized" for example, it seems there is no difference between quality = 1 and quality = 2...is it wrong?

thanks in advance for the answer.
Alberto

**Comment by fracape** [comment, marked as answer]
Hi,
You are correct that the term "quality" does not describe well the different pre-trained models. In traditional video coding, you would talk about R-D operating points. They correspond to a Lambda used in the rate-distortion cost criterion J=D +λR.
However, considering the result curves that we provide, the models show increasing reconstructed quality (PSNR/MS-SSIM) with the lambda (and "quality"). That can depend on a specific image content, but in general, you should get a higher quality and bitrate for q=2 than q=1.

**Reply by AlbertoPresta** [reply]
ok thanks! 
what values of lambda did you use for training?

**Reply by fracape** [reply]
please refer to the model zoo section of the doc: https://interdigitalinc.github.io/CompressAI/zoo.html
